front-end/src/Profile.css

```diff
@@ -153,4 +157,99 @@
   left: 0;
   bottom: 0;
   background-color: rgba(0,0,0,0.75);
+}
+
+#profile-show-container {
```

Comment:
I'm also not sure what's causing it, but on my screen, there's now no space between shows in the recently added shows display. [Here's an image](https://cdn.discordapp.com/attachments/776234836456898602/825188293527863336/Screenshot_20210326_220316.png).

Reply:
it works on my end and I sent screenshot on discord and slack -- are you viewing http://localhost:3006/profile/1 page?

Reply:
Yep, I'm viewing that URL. It might be because of another screen width thing, but I'm not sure.

Reply:
since last commit, I have updated the css query max width, can you please recheck again?

Reply:
yep, looks good!

Reply:
great!